docs/reusable/javascript-tracker-cross-domain/_index.md

```diff
@@ -0,0 +1,112 @@
+The JavaScript Tracker can add an additional parameter named “_sp” to the querystring of outbound links.
+This process is called “link decoration”.
+The `_sp` value includes the domain user ID for the current page and the time at which the link was clicked (according to the device's clock).
+This makes these values visible in the “url” field of events sent by an instance of the JavaScript Tracker on the destination page.
+The enrichment process will use these values to populate the `refr_domain_userid` and `refr_dvce_tstamp` fields for all events fired on the destination page where the URL includes the `_sp` parameter.
+
+You can configure which links get decorated this way using the `crossDomainLinker` field of the configuration object.
+The value should be a function taking one argument (the link element) and return `true` if the link element should be decorated and `false` otherwise.
+For example, this function would only decorate those links whose destination is “[http://acme.de](http://acme.de/)” or whose HTML id is “crossDomainLink”:
+
+```javascript
+{
+  crossDomainLinker: function (linkElement) {
+    return (linkElement.href === 'http://acme.de' || linkElement.id === 'crossDomainLink');
+  }
+}
+```
+
+If you want to decorate every link to the domain github.com:
+
+```javascript
+{
+  crossDomainLinker: function (linkElement) {
+    return /^https:\/\/github\.com/.test(linkElement.href);
+  }
+}
+```
+
+If you want to decorate every link, regardless of its destination:
+
+```javascript
+{
+  crossDomainLinker: function (linkElement) {
+    return true;
+  }
+}
+```
+
+Note that the above will decorate “links” which are actually just JavaScript actions (with an `href` of `"javascript:void(0)"`), or links to email addresses or telephone numbers ([`mailto:`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#linking_to_an_email_address) or [`tel:`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#linking_to_telephone_numbers) schemes).
+To exclude these links, check them explicitly:
+
+<>{ props.children[0] }</>
+
+Alternatively, only count links that parse as web URLs by checking the link's [`hostname`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLAnchorElement/hostname).
+This should automatically exclude links that don't lead simply to other web pages.
+
+<>{ props.children[1] }</>
```

Comment:
How does this work when children are in markdown? Each paragraph is a child? Would this break easily if someone unaware of this logic writes an extra paragraph? I’d wrap the text at the call site into two `<div>`s just for good measure.